src/styles/sidebar/components/focused-mode-header.scss

```diff
@@ -1,23 +1,27 @@
 @use "../../mixins/focus";
 @use "../../mixins/forms";
 @use "../../mixins/layout";
+@use "../../mixins/molecules";
 @use "../../mixins/utils";
 @use "../../variables" as var;
 
 // A dark grey button used for the primary action
 // in a form
 .focused-mode-header {
+  @include molecules.card-frame;
   @include layout.row($align: center);
-  @include utils.border;
   position: relative;
 
   background-color: var.$color-background;
```

Comment:
background-color is already set with  @include molecules.card-frame;

Reply:
Nice catch!